modules/touch/src/touchinteraction.cpp

```diff
@@ -552,9 +524,7 @@ void TouchInteraction::directControl(const std::vector<TuioCursor>& list) {
     }
 }
 
-// Traces the touch input into the scene and finds the surface coordinates of touched
-// planets (if occuring)
-void TouchInteraction::findSelectedNode(const std::vector<TuioCursor>& list) {
+void TouchInteraction::findSelectedNode(const std::vector<TouchInputs>& list) {
     //trim list to only contain visible nodes that make sense
     std::string selectables[30] = {
         "Sun", "Mercury", "Venus", "Earth", "Mars", "Jupiter", "Saturn", "Uranus",
```

Comment:
Are there any new nodes in master that we would want to add here @alexanderbock ?

Reply:
I opened an issue about this - see #1041